hooks/post-command

```diff
@@ -32,22 +41,12 @@ if [[ "${#CORE_DUMPS[@]}" > 0 ]]; then
                 (sleep "${DEBUGGER_TIMEOUT}"; kill "${DBG_PID}" 2>/dev/null >/dev/null; echo "DEBUGGER KILLED BY WATCHDOG TIMER") &
 
                 # Wait for one debugging command to finish before starting the next
-                wait "${DBG_PID}" || true
+                wait -f "${DBG_PID}" || true
             done
```

Comment:
If this async printing isn't reliable enough for you, we could also pipe this through dd like so:
```suggestion
            done | dd bs=64k status=none
```
such that all of the content printed here gets buffered via `dd` up to to the specified chunk size (64k) to make it atomic (and the same for each upload upon success of it as well)